doc/api/globals.md

```diff
@@ -629,6 +629,40 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency}`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
```

Comment:
Is this always the same as [`DefaultLocale()` from ECMA-402](https://tc39.es/ecma402/#sec-defaultlocale)? If so, it might be worth mentioning that this matches the default locale used by JavaScript APIs such as `Intl`.